LightFramework/src/rendering/scenerenderer.cpp

```diff
@@ -160,21 +218,20 @@ namespace Light {
 		m_framebuffer = framebuffer;
 	}
 
-
 	void SceneRenderer::renderOutline(std::shared_ptr<Scene> scene, Entity entity)
 	{
 		m_outlineFramebuffer->bind();
 		m_outlineFramebuffer->clearAttachment(0, 0);
 		m_outlineFramebuffer->clearDepthAttachment();
-		if(entity && entity.hasComponent<TransformComponent>() && entity.hasComponent<MeshComponent>())
+		if (entity && entity.hasComponent<TransformComponent>() && entity.hasComponent<MeshComponent>())
 		{
-			auto [transform, mesh]= scene->m_registry.get<TransformComponent, MeshComponent>((entt::entity)(uint32_t)entity);
+			auto [transform, mesh] = scene->m_registry.get<TransformComponent, MeshComponent>((entt::entity)(uint32_t)entity);
```

Comment:
Even though it had a C style cast before also, consider using a C++ style typecast here.

This line is pretty confusing. A comment explaining what is happening might be warranted here. Is this a double typecast?

Reply:
I'd not make any changes here iirc, this was originally present